# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../gnutls/gnutls-2.12.20-glibc-2.16-no-gets.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -up gnutls-2.12.20/gl/stdio.in.h.build gnutls-2.12.20/gl/stdio.in.h
--- gnutls-2.12.20/gl/stdio.in.h.build	2012-03-01 16:47:48.000000000 +0100
+++ gnutls-2.12.20/gl/stdio.in.h	2012-06-18 16:50:51.307755234 +0200
@@ -698,7 +698,7 @@ _GL_WARN_ON_USE (getline, "getline is un
 # endif
 #endif
 
-#if @GNULIB_GETS@
+#if 0
 # if @REPLACE_STDIO_READ_FUNCS@ && @GNULIB_STDIO_H_NONBLOCKING@
 #  if !(defined __cplusplus && defined GNULIB_NAMESPACE)
 #   undef gets
diff -up gnutls-2.12.20/lib/gl/stdio.in.h.build gnutls-2.12.20/lib/gl/stdio.in.h
--- gnutls-2.12.20/lib/gl/stdio.in.h.build	2012-03-01 16:53:13.000000000 +0100
+++ gnutls-2.12.20/lib/gl/stdio.in.h	2012-06-18 16:50:22.846115787 +0200
@@ -698,7 +698,7 @@ _GL_WARN_ON_USE (getline, "getline is un
 # endif
 #endif
 
-#if @GNULIB_GETS@
+#if 0
 # if @REPLACE_STDIO_READ_FUNCS@ && @GNULIB_STDIO_H_NONBLOCKING@
 #  if !(defined __cplusplus && defined GNULIB_NAMESPACE)
 #   undef gets
